core/src/test/java/com/linecorp/armeria/server/http/HttpServiceTest.java

```diff
@@ -91,12 +102,39 @@ protected void doGet(
                               res.respond(HttpStatus.NO_CONTENT);
                           }
                       }.decorate(LoggingService::new));
+            sb.service(PathMapping.ofPrefix("/dynamic"), new DynamicHttpServiceBuilder()
+                    .add(HttpMethod.POST, "/{var}", (ctx, req, arguments) -> {
+                             final DefaultHttpResponse res = new DefaultHttpResponse();
+                             final HttpData data = HttpData.ofUtf8(arguments.get("var"));
+                             final long current = System.currentTimeMillis();
+                             HttpHeaders headers = HttpHeaders.of(HttpStatus.OK)
+                                                              .setInt(HttpHeaderNames.CONTENT_LENGTH,
+                                                                      data.length())
+                                                              .setTimeMillis(HttpHeaderNames.DATE, current);
+                             res.write(headers);
+                             res.write(data);
+                             res.close();
+                             return res;
+                         }
+                    ).add(new ResponseStrategy())
+                    .build().decorate(LoggingService::new));
         } catch (Exception e) {
             throw new Error(e);
         }
         server = sb.build();
     }
 
+    public static class ResponseStrategy {
+        @RequestMapping(method = { HttpMethod.GET, HttpMethod.POST }, value = "/{message}/with/{count}")
```

Comment:
How about adding dedicated method annotations such as `@GET`, `@POST` instead for brevity? The resulting annotations could be:

```java
@GET @POST
@RequestMapping("/{message}/with/{count}")
```